.eslintrc.js

```diff
@@ -32,8 +32,10 @@ module.exports = {
 		"valid-jsdoc": ["error", {
 			"prefer": {
 				"return": "returns",
+				"prop": "property",
 				"memberof": "DONTUSE",
 				"class": "DONTUSE",
+				"extends": "DONTUSE",
```

Comment:
as @mohsen1 discovered in #7514, `@extends` is necessary when extending a generic type because Javascript doesn't have any way to specify the type argument, and the compiler can't infer an argument:

```js
/**
 * @template T
 * @extends {Set<T>}
 */
class SortableSet extends Set {
  // ...
}
```

Reply:
Maybe we keep it as "do not use" and override it if we're extending a generic type.